src/ui/common/src/components/primitives/LoadingButton.styles.tsx

```diff
@@ -2,7 +2,9 @@ import LoadingButton, { loadingButtonClasses } from '@mui/lab/LoadingButton';
 import { buttonClasses } from '@mui/material/Button';
 import { styled } from '@mui/material/styles';
 
-const AqueductLoadingButton = styled(LoadingButton)(({ theme }) => {
+import { theme } from '../../styles/theme/theme';
```

Comment:
Not sure if this is intentional, but `theme` is actually passed in by default by MUI. 

I think the idea is that if you wanted to have multiple nested themes rather than just 1, this would inherit the theme from the closest location.

Reply:
I actually got some compiler errors here and was having trouble getting this to work with common. I'm not really sure how we can go about typing it, but open to suggestions.